src/fides/api/models/privacy_experience.py

```diff
@@ -483,24 +493,10 @@ def new_experience_needed(
                 data={
                     "region": region,
                     "component": ComponentType.privacy_center,
-                    "delivery_mechanism": DeliveryMechanism.link,
                 },
             )
             added_experiences.append(privacy_center_experience)
 
-        # See if we need to update an existing Link Overlay to be delivered by Banner instead
-        if banner_delivery_update_needed(overlay_experience, overlay_notices):
```

Comment:
Removing some of this validation and behind-the-scenes efforts now that this requirement has been removed, as I'm updating PrivacyExperiences to make sure we always have the appropriate rows to back PrivacyNotices.